src/test/scala/scala/concurrent/stm/TMapSuite.scala

```diff
@@ -398,13 +398,12 @@ class TMapSuite extends FunSuite {
 
   test("view builder magic") {
     val fwd = TMap.View(1 -> "one", 2 -> "two")
-    val rev = fwd map { kv => (kv._2 -> kv._1) }
-    val rev2: TMap.View[String, Int] = rev
```

Comment:
This is a no-op line. Perhaps you wanted to test the type inferred for `rev`. If so, I could re-add the line, perhaps calling a 'test' method so `rev2` is referenced.